src/services/GraphDataSource.ts

```diff
@@ -216,32 +216,46 @@ export default class GraphDataSource {
 
   // Some helpers
 
-  public fetchForApp = (duration: DurationInSeconds, namespace: string, app: string) => {
+  public fetchForApp = (duration: DurationInSeconds, namespace: string, app: string, injectServiceNodes?: boolean) => {
     const params = GraphDataSource.defaultFetchParams(duration, namespace);
     params.graphType = GraphType.APP;
+    params.injectServiceNodes = !!injectServiceNodes;
```

Comment:
Minor nitpick: injectServiceNodes is already boolean, no need to do the double negative ;)

Reply:
another nitpick, it's boolean | undefined.